### PR TITLE
Bump examples/allocation-endpoint

### PR DESCRIPTION
**Pull request by Kalaiselvi84**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, press enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug

/kind cleanup

> /kind documentation
> /kind feature
> /kind hotfix
> /kind release

**What this PR does / Why we need it**:

**Which issue(s) this PR fixes**:

Closes #

**Special notes for your reviewer**:

### CODE REVIEW

File: examples/allocation-endpoint/terraform/variable.tf [hunk 1]
```diff
@@ -20,7 +20,7 @@ variable "project_id" {
 variable "ae_proxy_image" {
   type        = string
   description = "The docker image of the allocation proxy."
-  default     = "us-docker.pkg.dev/agones-images/examples/allocation-endpoint-proxy:0.5"
```
Comment (gongmax) [inline examples/allocation-endpoint/terraform/variable.tf:23]: The version here should only be bumped until the new image has been pushed to the registry.

Comment (Kalaiselvi84) [reply examples/allocation-endpoint/terraform/variable.tf:23]: This image has been successfully built and pushed to my project. Do we need to test it locally?🤔

Comment (Kalaiselvi84) [reply examples/allocation-endpoint/terraform/variable.tf:23]: @gongmax, if you approve this PR, I will update the latest tag in production🙂

Comment (gongmax) [reply examples/allocation-endpoint/terraform/variable.tf:23]: Well, ideally we should only update the version here after the new image has been pushed to agones registry. I.e., the steps should be: 1. bump the version in the makefile. 2. build and push the new image. 3. update the image version where it is used, eg this variable.tf file. 

I can approve this this time, but please push the image to agones registry ASAP.

**Review by gongmax — APPROVED** [review]

File: examples/allocation-endpoint/terraform/variable.tf (hunk 1 above)
Comment (Kalaiselvi84) [reply examples/allocation-endpoint/terraform/variable.tf:23]: It's done. https://screenshot.googleplex.com/QJjFMME7qVUxMEw

I was under the impression that I had to build and push after receiving approval. Thank you for clarifying.🙂

**Pull request merged**